qir/qir-tests/reduction_tests/inlining-input.ll

```diff
@@ -54,5 +54,5 @@ declare void @__quantum__rt__message(%String*)
 declare void @__quantum__rt__string_update_reference_count(%String*, i32)
 
 attributes #0 = { "InteropFriendly" }
-attributes #1 = { "EntryPoint" }
+attributes #1 = { "InteropFriendly" }
```

Comment:
I'm confused as to why both #0 and #1 attributes are now "InteropFriendly" instead of one being "EntryPoint" attributed. Doesn't the logic for matching the execution path only work when exactly one function with the chosen entry point attribute name exists?

Reply:
The #0 clearly should not be there as it is unused.